environment.yml

```
@@ -0,0 +1,22 @@
+name: vitessce-data
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python>=3.6
```

```suggestion
  - python>=3.7
```
Stable dict order isn't official until 3.7, and it's very useful.